.eslintrc

```diff
@@ -6,12 +6,14 @@
         "mocha": true,
         "jest": true
     },
-    "extends": "airbnb",
+    "extends": ["airbnb", "eslint:recommended"],
     "parser": "babel-eslint",
     "rules": {
         "indent": 2,
         "prefer-const": 1,
         "react/jsx-curly-spacing": [ 2, "always" ],
-        "react/jsx-filename-extension": 0
+        "react/jsx-filename-extension": 0,
+        "semi": "off",
```

Comment:
IMO, this rule should not be turn off. It should be either:

```
        "semi": [ "error", "always" ],
```

or

```
        "semi": [ "error", "never" ],
```

to enforce a consistent style across the codebase.

Reply:
That's my mistake! I didn't read the docs carefully. Thanks for your suggestion.